test/integration/apiserver/print_test.go

```diff
@@ -113,10 +113,6 @@ var kindWhiteList = sets.NewString(
 	"Eviction",
 	// --
 
-	// k8s.io/kubernetes/pkg/apis/componentconfig
-	"KubeSchedulerConfiguration",
-	// --
-
```

Comment:
:+1:

Reply:
little knowledge about this, and if not wrong, please correct me:slightly_smiling_face:

1. should we keep `KubeSchedulerConfiguration` here, just modify the description?
2.  i don't know wether we should add `kubeControllerManagerConfiguration` in here

Reply:
This test is about printing resources for the API server. A reference to any ComponentConfig types is wrong because they are not served by an API server. Hence I removed the old reference to the kube-scheduler config as I now found it.